### ISSUE TITLE
Analyse usage of `copyToClipboard` feature in the app and unify it if possible

### ISSUE THREAD

**Issue by noeliaSD** [status-im/status-desktop#13761]
### Description

We have different ways to do the same action `copyToClipboard` across the app. Here are listed some of the used paths but probably they can be more:
- `ui/imports/utils/Utils.qml` --> copyToClipboard(text) --> calls to `globalUtils` object.
- For each store we also have the same method and calls to `globalUtils` object:
   - `AppLayouts/stores/RootStore.qml`
   - `AppLayouts/Profile/stores/XXStore.qml`
   - `AppLayouts/Wallet/stores/RootStore.qml`
   - In shared stores
   - ...

1. Is there a technical reason why this backend access to `globalUtils.copyToClipboard` is repeated on each store object?
2. Can we just use a single one like the one in `ui/imports/utils/Utils.qml`?

**Comment by caybro** [comment]
Yeah, it should be sufficient to just call `Utils.copyToClipboard()` from anywhere

**Comment by jrainville** [comment]
Closing as it seems refactored already (uses signals and `ClipboardUtils.setText(text)`)
